From 861c994a0a2a8bd3e27b23ac73f73b4103c13199 Mon Sep 17 00:00:00 2001
From: Daniel Borkmann <dborkman@redhat.com>
Date: Mon, 25 Nov 2013 11:21:28 -0500
Subject: [redhat] configs: add CONFIG_NET_CLS_BPF entry

Message-id: <48210ddf9c12e79cd5d6004a085ac45863a4b625.1385378100.git.dborkman@redhat.com>
Patchwork-id: 71117
O-Subject: [RHEL7 PATCH net 2/2] redhat: add CONFIG_NET_CLS_BPF entry
Bugzilla: 958035
RH-Acked-by: David S. Miller <davem@redhat.com>
RH-Acked-by: Thomas Graf <tgraf@redhat.com>
RH-Acked-by: Jiri Pirko <jpirko@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=958035
Upstream Status: none
Brew: http://brewweb.devel.redhat.com/brew/taskinfo?taskID=6630079
Tested: verified on beaker w/ various BPF filters + monitoring w/ bmon

Add an entry for RH-specific Kconfig generator.

Signed-off-by: Daniel Borkmann <dborkman@redhat.com>
Signed-off-by: Jiri Benc <jbenc@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/configs/generic/CONFIG_NET_CLS_BPF b/redhat/configs/generic/CONFIG_NET_CLS_BPF
new file mode 100644
index 0000000..5645a27
--- /dev/null
+++ b/redhat/configs/generic/CONFIG_NET_CLS_BPF
@@ -0,0 +1 @@
+CONFIG_NET_CLS_BPF=m
-- 
1.7.1